src/components/layouts/FilterBar.tsx

```diff
@@ -14,19 +19,57 @@ import { IAssigneeCombined, View } from '@/types/interfaces'
 import { CrossIcon, FilterByAsigneeIcon } from '@/icons'
 import { ViewModeSelector } from '../inputs/ViewModeSelector'
 import { FilterByAssigneeBtn } from '../buttons/FilterByAssigneeBtn'
+import { Token } from '@/types/common'
+import FilterButtonGroup from '@/components/buttonsGroup/FilterButtonsGroup'
 
-export const FilterBar = ({ updateViewModeSetting }: { updateViewModeSetting: (mode: View) => void }) => {
+export const FilterBar = ({
+  updateViewModeSetting,
+  getTokenPayload,
+}: {
+  updateViewModeSetting: (mode: View) => void
+  getTokenPayload: () => Promise<Token>
+}) => {
   const [searchText, setSearchText] = useState('')
-
+  const [activeButtonIndex, setActiveButtonIndex] = useState<number>()
```

Comment:
Wouldn't it make more sense to add the default active index here? By default it should be the one for My Tasks

Reply:
Wouldn't a default for all task be more relevant?